modules/ocf_backups/files/prune-old-backups

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import json
+import shlex
+import subprocess
+import sys
+import textwrap
+
+RETENTION_TIME = datetime.timedelta(days=180) # 6 months
+
+BOX_FTP = 'ftps://ftp.box.com:990'
+BACKUP_PREFIX = 'ocf-backup-'
+
+LFTP_INIT = '''
+    set ftps:initial-prot ""
+    set ftp:ssl-force true
+    set ftp:ssl-protect-data true
+    open {box_ftp}
+    user {email} {passwd}
+'''
+
+def parse_date(dtstr):
+    """Converts a 'YYYY-MM-DD' formatted string to a datetime.date object."""
```

Comment:
does strptime("%Y-%m-%d") not work here?

Reply:
Yeah that's probably better.